escp_test.go

```
@@ -145,3 +363,213 @@ func TestIsDirEmpty(t *testing.T) {
+	remote, password, err := parseE2ESSHURL(rawURL)
+	if err != nil {
+		t.Fatalf("Invalid ESCP_E2E_SSH_URL: %v", err)
+	}
+
+	homeDir := t.TempDir()
+	sshDir := filepath.Join(homeDir, ".ssh")
+	if err := os.MkdirAll(sshDir, 0700); err != nil {
+		t.Fatalf("Create test .ssh directory: %v", err)
+	}
+
+	knownHosts, err := exec.Command("ssh-keyscan", "-p", remote.Port, remote.Host).Output()
```

golangci-lint (2.11.4)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)